commands/CloudCommands.js

```diff
@@ -210,7 +210,8 @@ CloudCommand.prototype = extend(BaseCommand.prototype, {
 					choices: [
 						'Photon',
 						'Core',
-						'P1'
+						'P1',
+						'Electron'
```

Comment:
Unless we have some "known apps" for the Electron added, there is no point to add it here.